Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -347,136 +350,157 @@ public void run() {
         }
 
         @Override
-        public void onUnableToPlayAd(@NonNull String placementReferenceId, String reason) {
+        public void onAdPlayError(@NonNull String placementId, VungleException exception) {
             MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onUnableToPlayAd - Placement ID: " +
-                    placementReferenceId + ", reason: " + reason);
-            if (mPlacementId.equals(placementReferenceId)) {
+                    placementId + ", reason: " + exception.getLocalizedMessage());
+            if (mPlacementId.equals(placementId)) {
+
                 mIsPlaying = false;
+                final MoPubErrorCode errorCode = mapErrorCode(exception.getExceptionCode());
+
                 mHandler.post(new Runnable() {
                     @Override
                     public void run() {
-                        if (mInteractionListener == null && mLoadListener != null) {
-                            mLoadListener.onAdLoadFailed(MoPubErrorCode.NETWORK_NO_FILL);
-                            MoPubLog.log(getAdNetworkId(), LOAD_FAILED, ADAPTER_NAME,
-                                    MoPubErrorCode.NETWORK_NO_FILL.getIntCode(),
-                                    MoPubErrorCode.NETWORK_NO_FILL);
-                        } else if (mInteractionListener != null) {
-                            mInteractionListener.onAdFailed(MoPubErrorCode.NETWORK_NO_FILL);
-                            MoPubLog.log(getAdNetworkId(), SHOW_FAILED, ADAPTER_NAME,
-                                    MoPubErrorCode.NETWORK_NO_FILL.getIntCode(),
-                                    MoPubErrorCode.NETWORK_NO_FILL);
+                        if (mInteractionListener != null) {
```

Comment:
Don't know why adding mLoadListener related code. For Interstitial ad, find the same issue in VungleInterstitial.java file as well.
Related PR: https://github.com/mopub/mopub-android-mediation/pull/257/commits/86d1c3e0f9469c32520d097f9456048ddb8fac1a#diff-aceb4d76c06d9382480e8f101358698c3e8c041110a4e1c7d93efd5d88d967a7R352-R362